music_assistant/providers/musiccast/constants.py

```diff
@@ -0,0 +1,80 @@
+"""Constants."""
+
+from music_assistant.constants import (
+    CONF_ENTRY_ENABLE_ICY_METADATA,
+    CONF_ENTRY_FLOW_MODE,
```

Comment:
Do you want to even offer flow mode ? You can also choose to completely hide the option (so default disabled) if you think that all MusicCast players handle the enqueueing just fine